rust/crd/src/security.rs

```diff
@@ -273,6 +280,7 @@ impl KafkaTlsSecurity {
         if let Some(tls_server_secret_class) = self.get_tls_secret_class() {
             // We have to mount tls pem files for kcat (the mount can be used directly)
             pod_builder.add_volume(Self::create_tls_volume(
+                &self.kafka.name_any(),
```

Comment:
The service name should be calculated by the according function:

```suggestion
                &self.kafka.broker_role_service_name().context(...)?,
```

Reply:
a37c54c